packages/serverpod/lib/src/web_server/web_server.dart

```diff
@@ -277,28 +277,28 @@ abstract class Route {
   }
 }
 
-/// A [ComponentRoute] is the most convenient way to create routes in your server.
-/// Override the [build] method and return an appropriate [Component].
-abstract class ComponentRoute extends Route {
-  /// Override this method to build your web [Component] from the current [session]
+/// A [WidgetRoute] is the most convenient way to create routes in your server.
+/// Override the [build] method and return an appropriate widget.
```

Comment:
It should be his right?

```suggestion
/// Override the [build] method and return an appropriate [WebWidget].
```